### ISSUE TITLE
New vue project following the installation guide fail to start

### ISSUE THREAD

**Issue by jackysee** [mozilla-neutrino/neutrino-dev#591]
Following exactly the installation guide ( https://neutrino.js.org/packages/vue/ ) to create a bare bone vue project, after `neutrino start` and open chrome it throws:

```
Uncaught DOMException: Failed to execute 'createElement' on 'Document': The tag name provided ('var disposed = false;
var normalizecomponent = require("!../node_modules/vue-loader/lib/component-normalizer");
/* script */
import __vue_script__ from "!!../node_modules/babel-loader/lib/index.js?{\"cachedirectory\":true,\"plugins\":[[\"c:\\\\workspace\\\\web\\\\neutrino-test\\\\node_modules\\\\fast-async\\\\plugin.js\",{\"spec\":true}],\"c:\\\\workspace\\\\web\\\\neutrino-test\\\\node_modules\\\\babel-plugin-syntax-dynamic-import\\\\lib\\\\index.js\"],\"presets\":[[\"c:\\\\workspace\\\\web\\\\neutrino-test\\\\node_modules\\\\babel-preset-env\\\\lib\\\\index.js\",{\"debug\":false,\"modules\":false,\"usebuiltins\":true,\"exclude\":[\"transform-regenerator\",\"transform-async-to-generator\"],\"targets\":{\"browsers\":[\"last 2 chrome versions\",\"last 2 firefox versions\",\"last 2 edge versions\",\"last 2 opera versions\",\"last 2 safari versions\",\"last 2 ios versions\"]}}]]}!../node_modules/vue-loader/lib/selector?type=script&index=0&bustcache!./app.vue";
/* template */
import __vue_template__ from "!!../node_modules/vue-loader/lib/template-compiler/index?{\"id\":\"data-v-04c2046b\",\"hasscoped\":false,\"buble\":{\"transforms\":{}}}!../node_modules/html-loader/index.js?{\"name\":\"[name].[ext]\"}!../node_modules/vue-loader/lib/selector?type=template&index=0&bustcache!./app.vue";
/* template functional */
var __vue_template_functional__ = false;
/* styles */
var __vue_styles__ = null;
/* scopeid */
var __vue_scopeid__ = null;
/* moduleidentifier (server only) */
var __vue_module_identifier__ = null;
var component = normalizecomponent(__vue_script__, __vue_template__, __vue_template_functional__, __vue_styles__, __vue_scopeid__, __vue_module_identifier__);
component.options.__file = "src\\app.vue";
if (component.esmodule && object.keys(component.esmodule).some(function (key) {
  return key !== "default" && key.substr(0, 2) !== "__";
})) {
  console.error("named exports are not supported in *.vue files.");
}

/* hot reload */
if (module.hot) {
  (function () {
    var hotapi = require("vue-hot-reload-api");
    hotapi.install(require("vue"), false);
    if (!hotapi.compatible) return;
    module.hot.accept();
    if (!module.hot.data) {
      hotapi.createrecord("data-v-04c2046b", component.options);
    } else {
      hotapi.reload("data-v-04c2046b", component.options);
      ' + ';
    }
    module.hot.dispose(function (data) {
      disposed = true;
    });
  })();
}

export default component.exports;') is not a valid name.
    at Object.isUnknownElement (webpack-internal:///./node_modules/vue/dist/vue.runtime.esm.js:5166:21)
    at isUnknownElement$$1 (webpack-internal:///./node_modules/vue/dist/vue.runtime.esm.js:5419:14)
    at createElm (webpack-internal:///./node_modules/vue/dist/vue.runtime.esm.js:5438:13)
    at Vue$3.patch [as __patch__] (webpack-internal:///./node_modules/vue/dist/vue.runtime.esm.js:5982:9)
    at Vue$3.Vue._update (webpack-internal:///./node_modules/vue/dist/vue.runtime.esm.js:2612:19)
    at Vue$3.updateComponent (webpack-internal:///./node_modules/vue/dist/vue.runtime.esm.js:2740:10)
    at Watcher.get (webpack-internal:///./node_modules/vue/dist/vue.runtime.esm.js:3090:25)
    at new Watcher (webpack-internal:///./node_modules/vue/dist/vue.runtime.esm.js:3079:12)
    at mountComponent (webpack-internal:///./node_modules/vue/dist/vue.runtime.esm.js:2747:3)
    at Vue$3.$mount (webpack-internal:///./node_modules/vue/dist/vue.runtime.esm.js:7828:10)
```

On Node 8.6.0 / yarn 1.1.0

**Comment by eliperelman** [comment]
Thanks for the report! Sorry about that, I have issued a fix in #599, which was released in v8.0.11. Please reopen if this does not resolve your issue. Thanks!

**Comment by jackysee** [comment]
Thanks. I also tried the `yarn create @neutrinojs/project` approach, but it's another problem. I will create another issue.
